README.md

```diff
@@ -38,40 +38,62 @@ Are you missing an example? Please feel free to [open an issue](https://github.c
 
 <!-- Please keep the absolute URLs so it's easier to copy&paste to prisma/prisma/README.md  -->
 
-## TypeScript
+## Prisma Data Platform
+
+The [`prisma-data-platform`](./prisma-data-platform/) folder contains examples of projects using products offered by the [Prisma Data Platform](https://console.prisma.io/).
```

Comment:
We don't generally talk about Accelerate and Pulse being "offered by" the Prisma Data Platform.

How about skipping the PDP part and just mentioning Accelerate and Pulse?

Reply:
Can't add a comment on that line, but the "About this repository" seems outdated now too, with having multiple products in this repo, or is that still applicable?